tests/bson/bson-document-property-access-002.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+MongoDB\BSON\Document property access (coalesce operator)
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/basic.inc';
+
+$document = MongoDB\BSON\Document::fromPHP([]);
+
+var_dump(isset($document->foo));
+var_dump($document->foo ?? 'fallback');
+
+?>
+===DONE===
```

Comment:
Why do you need this `==DONE==`?

Reply:
Is this a general question or specific to this test? Generally, I can tell you that `==DONE==` has been used to separate test output from possible memory leaks info when running tests on debug version of PHP. That's also why we manually call `exit(0)`.